torch/testing/_internal/common_methods_invocations.py

```diff
@@ -20789,6 +20789,10 @@ def __init__(
     #
     # Elementwise Ternary Reference OpInfos
     #
+    PythonRefInfo(
+        "_refs.addcdiv",
```

Comment:
It may be interesting to add extra reference samples that test for:
- Broadcasting
- Casting with respect to `value`

Reply:
- broadcasting is already covered by sample inputs
- promotion seems broken, so i'll try another attempt to land this one first: https://github.com/pytorch/pytorch/pull/74234

Reply:
Land, or simply make the function more restrictive, meaning: Just check that the dtypes of everything are the same in both the C++ implementation and the ref implementation and roll with it.